en/mulesoft/modules/ROOT/pages/ms_composer_invocable_flows.adoc

```diff
@@ -0,0 +1,263 @@
+.. Optionally, to change the input fields, click *Add Input Fields* to open the *Sample JSON Editor* to edit the input fields.
++
+. Complete the flow. For example, configure system actions for the flow, define variables, or create flow controls. For more information, refer to xref:ms_composer_flows.adoc[Build a Composer Flow]. 
+. Click *Save* to save your flow.
+
+If you need to delete an invocable step from your flow: 
+
+. Access the flow containing the invocable flow step that you want to delete. 
+. In the left corner of the step, click *…* > *Delete step*. +
+The step is deleted.
+ 
+[[test-your-invocable-composer-flow]]
+=== Test Your Invocable Composer Flow 
+
+To test your invocable Composer flow: 
+
+. In your invocable flow, click *Test*. +
+If you have added input fields, the *Test Flow* window appears.
+. Enter a sample value for the input field or review the complex JSON to test the flow.
+. Click *Test Flow*. +
+The testing output appears. Using this output, you can verify if the flow returns the expected values in the sample output of the main canvas. 
+
+
+[[activate-your-invocable-composer-flow]]
+=== Activate Your Invocable Composer Flow 
+
+To activate your invocable Composer flow: 
+
+. In your invocable flow, click *Activate*. +
+The flow details page appears with a message indicating that the flow is now active. 
+. Click *Flow Credentials* to view the flow data that is used to invoke the flow from the external system.
+. Optionally, you can reset the flow credentials periodically to ensure continuous security. In the *Flow Credentials* window, click *Reset Client Secret*. +
+The *Client Secret* field is updated. You can click the clipboard icon next to the field to copy the new *Client Secret* value. 
++
+Once the client secret is reset, the previous client secret no longer works. You must manually share the new client secret to external systems that invoke this Composer flow. 
+. Ensure that the following fields have values: 
+* *URL*: The unique URL of the flow you are invoking. 
+* *Client ID*: The username or unique identifier to authenticate this flow.
+* *Client Secret*: The password or secret to authenticate this flow. 
+* *OpenAPI Specification*: The OpenAPI specification that describes the structure of the underlying API of this flow.
+. Click *Close*. 
+
+[[invoke-the-composer-flow-from-an-external-system]]
+== Invoke the Composer Flow from an External System
+
+After creating an invocable flow in Composer, you can invoke it from an external system.
+
+=== Invoke a Composer Flow from Another Composer Flow 
+
+You can create an invocable flow in Composer and invoke it from another Composer flow.
+
+To invoke a Composer flow from another Composer flow:
+
+. Access Composer.
+. xref:ms_composer_flows.adoc#create-a-new-flow[Create a new flow from scratch].
+. Click *+* and click *System Action*. Select *HTTP*.
```

Comment:
Does this work? Clic *+*>*System Action* and select *HTTP*.

Reply:
Yes!